service/src/main/java/io/camunda/service/GroupServices.java

```diff
@@ -123,18 +123,18 @@ public CompletableFuture<GroupRecord> deleteGroup(final long groupKey) {
   }
 
   public CompletableFuture<GroupRecord> assignMember(
-      final long groupKey, final long memberKey, final EntityType memberType) {
+      final String groupId, final String username, final EntityType memberType) {
     return sendBrokerRequest(
-        BrokerGroupMemberRequest.createAddRequest(groupKey)
-            .setMemberKey(memberKey)
+        BrokerGroupMemberRequest.createAddRequest(groupId)
+            .setMemberId(username)
             .setMemberType(memberType));
   }
 
   public CompletableFuture<GroupRecord> removeMember(
-      final long groupKey, final long memberKey, final EntityType memberType) {
+      final String groupId, final String username, final EntityType memberType) {
```

Comment:
🔧 And here 😄

Reply:
There is not DTO, groupId and username are coming from the path